.tekton/jupyter-minimal-ubi9-python-3-11-pull-request.yaml

```diff
@@ -10,12 +10,13 @@ metadata:
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/cancel-in-progress: "true"
     pipelinesascode.tekton.dev/max-keep-runs: '3'
-    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch == "main" && ( "jupyter/minimal/ubi9-python-3.11/Pipfile.lock".pathChanged()
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
+      == "main" && ( "jupyter/minimal/ubi9-python-3.11/Pipfile.lock".pathChanged()
```

Comment:
can't find a line length limit that would leave this unchanged ;(

Reply:
I see... well, let's leave this as is for now. We can bother with it later if we'll see that the automatic konflux updates will be fighting with us about this.